### PR TITLE
Refactor and reuse "Actions" in "Moderate" and "Community"

### PR DESCRIPTION
**Pull request by alinhle**
## What does this PR do?
- Refactor `SuspendUserDialog` and the `BanUserDialog` into Top-Level Containers and Components.
- Reuse same logic across `Moderation` and `Community` tab.
- Rename previous `SuspendUserDialog` and related components of  `Community` to `RejectUsernameDialog`.

## How do I test this PR?

- Actions dropbox must appear in the `Flagged User` view of  `Community`
- Verify that the Ban, Suspend and RejectUsername workflow functions as expected.

### CODE REVIEW

File: locales/es.yml [hunk 1]
```diff
@@ -353,35 +354,32 @@ es:
...
-    email_message_suspend: "Querida {0}, De acuerdo a la guía comunitaria de {1}, su cuenta ha sido temporalmente suspendida. Durante este tiempo, no podrá comentar, marcar o involucrarse con otros comentaristas. Por favor unirse a la conversación {2}."
-    error_email_message_empty: "Debe especificar un mensaje de correo."
+    select_duration: "Seleccionar la duración de la suspensión"
+    one_hour: "1 hora"
     hours: "{0} horas"
-    no_cancel: "No cancelar"
+    days: "{0} días"
+    cancel: "Cancelar"
+    suspend_user: "Suspender Usuario"
+    email_message_suspend: "Querida {0}, De acuerdo a la guía comunitaria de {1}, su cuenta ha sido temporalmente suspendida. Durante este tiempo, no podrá comentar, marcar o involucrarse con otros comentaristas. Por favor unirse a la conversación {2}."
```
Comment (okbel) [inline locales/es.yml]: Maybe "Querido/a" would be best? 🤔

Comment (okbel) [reply locales/es.yml]: Or just the Username

**Review by okbel — APPROVED** [review]

File: locales/es.yml (hunk 1 above)
Comment (alinhle) [reply locales/es.yml]: Quizás, Quizás..

Comment (alinhle) [reply locales/es.yml]: Listo

Comment (okbel) [reply locales/es.yml]: Danke, liebe Kiwi

**Comment by okbel** [comment]
LGTM! :100:

File: locales/es.yml (hunk 1 above)
Comment (alinhle) [reply locales/es.yml]: Liebe/r Kiwi :-D

**Pull request merged**